src/components/CategoriesList.vue

```diff
@@ -1,15 +1,8 @@
 <template>
-	<NcAppNavigationItem
-		:title="title"
-		class="app-navigation-noclose separator-below"
-		:class="{ 'category-header': selectedCategory !== null }"
-		:open.sync="open"
-		:allow-collapse="true"
-		@click.prevent.stop="onToggleCategories"
-	>
-		<FolderIcon slot="icon" :size="20" />
+	<Fragment>
 		<NcAppNavigationItem
 			:title="t('notes', 'All notes')"
+			:class="{ active: null === selectedCategory }"
```

Comment:
Shouldn't it be `selectedCategory === null` ?

Reply:
Yep, more what the current code base and Nextcloud apps are using.